appengine/pubsub/app.js

```diff
@@ -70,9 +74,37 @@ app.post('/', formBodyParser, async (req, res, next) => {
 // [END gae_flex_pubsub_index]
 
 // [START gae_flex_pubsub_push]
-app.post('/pubsub/push', jsonBodyParser, (req, res) => {
+app.post('/pubsub/push', jsonBodyParser, async (req, res) => {
```

Comment:
Keep the old code in this region tag `gae_flex_pubsub_push` because it's used in the App Engine docs as a simple example. 

Create a new block with a new region tag `gae_flex_pubsub_auth_push` because we will use it in the Pub/Sub docs to show this beta feature. This will require setting up two different push endpoints.

Reply:
Done!